src/sdk/services/oauthService.ts

```diff
@@ -2,14 +2,12 @@ import { ManifestApi } from '../../manifest/ManifestApi';
 import runtime from '../RuntimeContext';
 
 class OAuthService {
```

Comment:
`OAuthService` is missing tests. 
Maybe we should create some, especially when we're doing big changes like this one.

Reply:
Added couple of tests